src/components/Cart/Cart.jsx

```diff
@@ -0,0 +1,105 @@
+import { Header } from "../Header/Header";
+import { NavPhone } from "../NavPhone/NavPhone";
+import { useCart, useNavPhone, useWishlist } from "../../contexts";
+import { useWindowSize } from "../../hooks";
+import "./cart.css";
+import EmptyCart from "../../images/empty_cart.svg";
+
+export const Cart = () => {
+  const { cart, cartDispatch } = useCart();
+  const { wishlistDispatch } = useWishlist();
+
+  const screenWidth = useWindowSize().width;
+  const { navPhoneVisible } = useNavPhone();
+
+  const getTotalCartQuantity = () =>
+    cart.reduce((acc, curr) => acc + curr.quantity, 0);
+  const getTotalCartPrice = () =>
+    cart.reduce((acc, curr) => acc + curr.price * curr.quantity, 0);
+
+  return (
+    <div className="container-app">
+      {screenWidth < 768 && navPhoneVisible && <NavPhone active="" />}
+      <Header active="" />
+      {cart.length !== 0 && (
+        <>
+          <div className="page-heading">Cart</div>
+          <div className="container-cart">
+            <div className="items-list">
+              {cart.map((item) => (
+                <div className="cart-card">
+                  <img
+                    className="cart-image"
+                    src={item.coverImage}
+                    alt={item.name}
+                  />
+                  <div className="cart-item-details">
+                    <div className="details">
+                      <div>
+                        <div className="name">{item.name}</div>
+                        <div className="subheading">
+                          <span>Platforms:</span> {item.platforms.join(", ")}
+                        </div>
+                      </div>
+                      <div className="cart-actions">
+                        {item.category === "peripheral" && (
+                          <span>quantity |</span>
+                        )}
+                        <span
+                          onClick={() =>
+                            cartDispatch({
+                              type: "REMOVE_FROM_CART",
+                              payload: item.id,
+                            })
+                          }
+                        >
+                          Delete
+                        </span>{" "}
+                        |{" "}
+                        <span

```

Comment:
Your components can be broken down into smaller components. Rather than having all the JSX for a component in one place. Would make the code more redable